PyStableMotifs/Reduction.py

```diff
@@ -935,6 +1004,43 @@ def generate_attr_dict(self):
                 # attr_list.append(node_state_dict)
             return attr_list
 
+    def simple_generate_attr_dict(self, MPBN_update=False):
```

Comment:
Can we merge this function into `generate_attr_dict` instead (which would take `MPBN_update` as an argument)? The MPBN_update argument and assertion in this function is a little unusual. While we're at it, we might want to rename `generate_attr_dict` to `generate_attr_dict_list`.

Reply:
I merged them and changed the name to generate_attr_dict_list.
I also fixed the name where the function is called